src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -755,6 +755,9 @@ internal unsafe struct MethodTable
         private const uint enum_flag_HasTypeEquivalence = 0x02000000;
 #endif // FEATURE_TYPEEQUIVALENCE
         private const uint enum_flag_HasFinalizer = 0x00100000;
+#if FEATURE_64BIT_ALIGNMENT
+        private const uint enum_flag_RequiresAlign8 = 0x00800000; // Type requires 8-byte alignment (only set on platforms that require this and don't get it implicitly)
```

Comment:
This bit can be checked in the FCall. we do not need to push it into CoreLib to make this work.